docs/tests.md

```diff
@@ -86,15 +95,12 @@ With the Simpletest UI apparently going away in Drupal 8.2 (cf [#2566767],
 The typical full command to run tests looks like this (`\` is to avoid too long a line):
 
     MONGODB_URI=mongodb://somemongohost:27017 \
-    SIMPLETEST_DB=mysql://someuser:somepassword@localhost/somedatabase \
-    phpunit -c core/phpunit.xml.dist
+    PHPUNIT_OPTS="-c phpunit.xml -v --debug --coverage-clover=modules/contrib/$MODULE_NAME/$COVERAGE_FILE"
```

Comment:
Now that we have a Functional test in the mongodb_watchdog module (in the yet unmerged phpunit branch) we also need SIMPLETEST_BASE_URL and SIMPLETEST_DB. And BROWSERTEST_OUTPUT_DIRECTORY is recommended to get test output.

Reply:
Note that BROWSERTEST_OUTPUT_DIRECTORY isn't sufficient on its own : the directory must exist and be writable, AND the phpunit.xml configuration needs to add `printerClass="\Drupal\Tests\Listeners\HtmlOutputPrinter"` to the top-level `phpunit` XML element in the `phpunit.xml` file.